sw/device/tests/rstmgr_alert_info_test.c

```diff
@@ -83,7 +83,7 @@ enum {
   kRoundOneDelay = 100,           // us
   kRoundTwoDelay = 100,           // us
   kRoundThreeDelay = 1000,        // us
-  kRoundFourDelay = 2500,         // us
+  kRoundFourDelay = 2800,         // us
```

Comment:
What's going on here? Does this test use I2C and did these changes slow it down or did something acidentally get added to the commit?